.storybook/stories/AuthenticatedAppShell/Menu.tsx

```diff
@@ -0,0 +1,176 @@
+import { forwardRef, useCallback, useRef, useState } from 'react';
+import { usePopper } from 'react-popper';
+import { useClickOutside, useTernaryState } from '@ag.ds-next/react/core';
+import { Button } from '@ag.ds-next/react/button';
+import { Box, Stack } from '@ag.ds-next/react/box';
+
+const MenuButton = ({ children, onClick }) => (
+	<Button variant="secondary" onClick={onClick}>
+		{children}
+	</Button>
+);
+
+const MenuList = forwardRef(({ children, ...props }, ref) => {
```

Comment:
This should follow https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-links/